web/components/logos/TopBar.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+import topbar_logo from "../../public/topbar_logo.png";
+
+interface TopBarProps {
+    title: string;
+    onNavToggle: () => void;
+}
+  
+const TopBar: React.FC<TopBarProps> = ({ title, onNavToggle }) => {
+    return (
+      <div className="bg-pink-600 text-white flex justify-between items-center p-4 z-40">
+        <button onClick={onNavToggle} className="text-white focus:outline-none lg:hidden">
+            <Image className="static px-10" src={topbar_logo} />
```

Comment:
The uploaded logos and icons should be SVGs instead of PNGs to avoid blurriness on larger screens, they can be downloaded from the Figma in Dev mode!